hbase-shell/src/main/ruby/shell/commands/get_largelog_responses.rb

```diff
@@ -0,0 +1,78 @@
+#
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements. See the NOTICE file distributed with this
+# work for additional information regarding copyright ownership. The ASF
+# licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Retrieve latest large log responses maintained in memory by RegionServers
+
+module Shell
+  module Commands
+    # Retrieve latest large log responses
+    class GetLargelogResponses < Command
+      def help
+        <<-EOF
+Retrieve latest LargeLog Responses maintained by each or specific RegionServers.
+Specify '*' to include all RS otherwise array of server names for specific
+RS. A server name is the host, port plus startcode of a RegionServer.
+e.g.: host187.example.com,60020,1289493121758 (find servername in
+master ui or when you do detailed status in shell)
+
+Provide optional filter parameters as Hash.
```

Comment:
This can probably be simplified too with 'TYPE' => LARGE_LOGS

Reply:
Same as above. Let's have type filter for server side logic but have clean, separate shell command and Admin API. Anyways, code would not be too much and would be simple for client.